src/Microsoft.Diagnostics.EventFlow.Outputs.EventHub/EventDataExtensions.cs

```diff
@@ -49,7 +47,7 @@ public static MessagingEventData ToMessagingEventData(this EventData eventData,
                 : ConvertToShoeboxTrace(eventData);
 
             // If this turns out to consume significant CPU time, we could serialize the object "manually".
-            // See https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Extensions.Logging.EventSource/EventSourceLogger.cs for an example.
+            // See https://github.com/dotnet/runtime/blob/0ea3b7d8cf957508a94061b8e921b28d6f6becd6/src/libraries/Microsoft.Extensions.Logging.EventSource/src/EventSourceLogger.cs for an example.
```

Comment:
```suggestion
            // See https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Logging.EventSource/src/EventSourceLogger.cs for an example.
```

Reply:
Why though? This is permalink that will 
"always" point to a code that is references in comment. Having `main` will point to "latest" which may not have code that is references in the comment?

Reply:
You are right--it is more important to refer to code that shows the example than to have a reference to current code.